### ISSUE TITLE
PostgreSQL transaction warnings when using SODAR Taskflow

### ISSUE THREAD

**Issue by mikkonie** [bihealth/sodar-docker-compose#17]
In the output of the Docker Compose network, I've noticed a flood of `there is no transaction in progress` errors when SODAR Taskflow is busy with some operation. Never seen this before. Need to investigate what could cause this and whether it's an actual issue.

I was thinking it might have something to do with how `python-irodsclient` uses Postgres. However, in that case we should get similar errors from using the same API from within SODAR as well, right?

Example:

```
postgres_1        | 2022-06-08 07:24:03.329 UTC [1679] WARNING:  there is no transaction in progress
postgres_1        | 2022-06-08 07:24:03.390 UTC [1688] WARNING:  there is no transaction in progress
postgres_1        | 2022-06-08 07:24:03.448 UTC [1679] WARNING:  there is no transaction in progress
postgres_1        | 2022-06-08 07:24:03.499 UTC [1688] WARNING:  there is no transaction in progress
postgres_1        | 2022-06-08 07:24:03.553 UTC [1679] WARNING:  there is no transaction in progress
postgres_1        | 2022-06-08 07:24:03.609 UTC [1688] WARNING:  there is no transaction in progress
```

This appears especially during `BatchMoveDataObjectsTask`. I also see this same thing in production so it's not limited to dev env use.

It's possible that the ongoing (but sadly postponed) integration of SODAR Taskflow into SODAR would fix this issue.

**Comment by mikkonie** [comment]
I've yet to pin down the root cause of this, but I have observed that some iRODS operations triggered from SODAR cause these warnings, while others do not.

The `data_delete` flow is one which definitely makes them happen, so I can use that as a starting point to debug this.

Also certain SODAR test cases can be used to reproduce this.

A temporary workaround would be to set up Postgres with `log_min_messages = error`, but I'd prefer to get rid of these warnings for real instead.

**Comment by mikkonie** [comment]
This actually looks to be [an open issue in iRODS itself](https://github.com/irods/irods/issues/5733). Not sure why I never thought of checking there?

Before they fix it, I guess disabling warnings is the way to go.

**Comment by mikkonie** [comment]
Fixed in `upgrade-4.3` by suppressing warnings, to be merged.

**Comment by mikkonie** [comment]
Done.
